Add Kong Gateway Operator docs (#6144)
Kong/docs.konghq.com: app/_src/gateway-operator/faq.md

---
title: FAQ
---

> **How can I use the Kong Gateway operator?**

Reviewer: ```suggestion
**How can I use the {{site.kgo_product_name}}?**
```

> The {{site.kgo_product_name}} can be installed by following the instructions at https://docs.konghq.com/gateway-operator/latest/install/

Reviewer 2: ```suggestion
You can install the {{site.kgo_product_name}} by following the instructions at https://docs.konghq.com/gateway-operator/latest/install/.
```

> The quickest way to get started is to attach your data planes to Konnect. See [getting started with Konnect](/gateway-operator/{{ page.release }}/get-started/konnect/install/) for more information

Reviewer 2: ```suggestion
The quickest way to get started is to attach your data planes to Konnect. For more information, see the [getting started with Konnect](/gateway-operator/{{ page.release }}/get-started/konnect/install/). 
```

---

> **How can I get help with the Kong Gateway Operator?**

Reviewer: ```suggestion
**How can I get help with the {{site.kgo_product_name}}?**
```
